tools/assetpipelineeditor/installer.iss.in

```diff
@@ -0,0 +1,43 @@
+; Script generated by the Inno Script Studio Wizard.
+; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
+
+[Setup]
+; NOTE: The value of AppId uniquely identifies this application.
+; Do not use the same AppId value in installers for other applications.
+; (To generate a new GUID, click Tools | Generate GUID inside the IDE.)
+AppId={{8A65FC8C-4309-421D-8F96-CECF6A6D8B50}
+AppName=Kuesa Asset Pipeline Editor
+AppVersion=1.0
```

Comment:
maybe it could be useful to use the QMake [VERSION](https://doc.qt.io/qt-5/qmake-variable-reference.html#version) variable here ?

Reply:
done